src/main.rs

```diff
@@ -102,8 +103,10 @@ fn build_response_json(people_present: Option<u32>, raspi_temperature: Option<f3
 }
 
 fn status_endpoint(_: Request, res: Response<Fresh>) {
-    let mut res = res.start().unwrap();
+    // Move response into mutable variable
+    let mut res = res;
```

Comment:
Yeah the position of the mut can be confusing if one thinks it belongs to the type. But it has to be like that, otherwise one couldn't declare a binding mutable and still let the compiler deduce the type.